client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
>  It should be just as simple to move the arrow to beginning of the line, instead of the end, to fix the jumping button problem.

Change it now to revert later. Makes no sense to me.

> I do think that having clickable arrow in nickname row as it was before is not a very good solution, as it may be confusing and not that obvious that it does something unique compared to other icons there.

Yeah, but `cursor: pointer` is savior here. With the whole line clickable it's hard not to notice something special about that line :smirk_cat: 

It may be the good idea to completely remove that icon and add something big and obvious here: 
![28764196-7e9ae1d4-75cc-11e7-8940-12be4be232f1](https://user-images.githubusercontent.com/3627488/29117356-8eb8cb8a-7d06-11e7-8106-fee24659a0cd.png)
because it's the safest spot for click action.